### PR TITLE
Feature/fork etrog

### PR DESCRIPTION
**Pull request by krlosMata**
This PR does the following:
- implements all functionalities related with `fork-etrog`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ on:
     branches:
       - main
       - develop
+      - feature/fork-etrog
```
Comment (ignasirv) [inline .github/workflows/main.yml:15]: No need now

File: package.json [hunk 2]
```diff
@@ -35,7 +37,7 @@
   },
   "homepage": "https://github.com/0xPolygonHermez/zkevm-commonjs#readme",
   "devDependencies": {
-    "@0xpolygonhermez/zkevm-contracts": "github:0xPolygonHermez/zkevm-contracts#v3.0.0-rc.1-fork.6",
+    "@0xpolygonhermez/zkevm-contracts": "github:0xPolygonHermez/zkevm-contracts#feature/l1-info-tree",
```
Comment (ignasirv) [inline package.json:40]: Do we have tag?

**Review by ignasirv — APPROVED** [review]
💯

File: package.json (hunk 2 above)
Comment (krlosMata) [reply package.json:40]: tests will fail since we need still to migrate to new bridge LxLy

**Pull request merged**